test/interface-tests.spec.js

```diff
@@ -564,4 +683,38 @@ describe('kubo-rpc-client tests against go-ipfs', () => {
   tests.stats(commonFactory)
 
   tests.swarm(commonFactory)
+}
+
+describe('kubo-rpc-client tests against kubo', async function () {
+  const goVersionsToTest = ['current']
+  /**
+   * Enable running the tests across all go-ipfs versions listed below.
+   * Works best with `--bail=false` otherwise the tests will stop at the first
+   * failure.
+   */
+  if (process.env.GO_IPFS_GAMUT != null) {
+    goVersionsToTest.push(...['12.0', '12', '13', '14', '15', '16'])
+  }
+
+  for (const version of goVersionsToTest) {
+    let ipfsBin
+    try {
+      const { path } = await import(`go-ipfs-${version}`)
+      ipfsBin = path()
+    } catch {
+      ipfsBin = undefined
+    }
+
+    const commonFactory = factory({
+      type: 'go',
+      ipfsBin
+    }, {
+      go: {
+        ipfsBin
+      }
+    })
+    describe(`go-ipfs ${version}${ipfsBin != null ? `at ${ipfsBin}` : ''}`, () => {
+      executeTests(commonFactory)
+    })
+  }
 })
```

Comment:
I added this as a way to discover if some version of kubo (go-ipfs to be exact) broke things in the kubo-rpc-client or tests.